### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Certificate_Screen Reader: NVDA is reading "Clickable" along with the list items present in the grid cell

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#205]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Google Chrome Version 68.0.3440.106 
Screen Reader: NVDA
**MAS ID Violated** : MAS1.3.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Web Server" and press enter.
5) Tab till Certificates and press enter
6) Start Screen Reader (Narrator/NVDA).
8) Verify screen reader is reading all the contents, controls, links, images properly throughout the process.

**Actual Result:**
NVDA is reading "Clickable" along with the list items when the user press the shortcut key "I" to navigate to the list items present in the cell. 
  
**Expected Result:**
NVDA should read "Clickable" for actionable items but here the list items are not actionable.
It should not read the list items as Clickable cause on pressing Enter on the list items no action is happening inside the grid. Either the list items should be interactive or NVDA should not read the list items as Clickable 

**User Impact:** 
Announcing "Clickable" for the list items may create confusion to the Visually Impaired Users or Screen Reader users. They might think the list items as actionable 

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Uploading Bug_205 _NVDA is reading Clickable along with the list items.pptx…]()

**Comment by jhkimnew** [comment]
This issue is gone with the recent product change.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
